### PR TITLE
Adds bear plushies

### PR DESCRIPTION
**Pull request by SubberTheFabulous**
Adds brand new bear plushies and literally nothing else.
image
I blame Prate.

### CODE REVIEW

File: code/game/objects/random/random.dm [hunk 1]
```diff
@@ -317,6 +317,8 @@
 		/obj/item/toy/plushie/kitten,
 		/obj/item/toy/plushie/lizard,
 		/obj/item/toy/plushie/farwa,
+		/obj/item/toy/plushie/bear
```
Comment (skull132) [inline code/game/objects/random/random.dm]:
```suggestion
		/obj/item/toy/plushie/bear,
```

File: code/game/objects/random/random.dm [hunk 2]
```diff
@@ -317,6 +317,8 @@
...
+		/obj/item/toy/plushie/bearfire
```
Comment (skull132) [inline code/game/objects/random/random.dm]:
```suggestion
		/obj/item/toy/plushie/bearfire,
```

**Comment by skull132** [comment]
Also, don't forget to use `!review` when the PR is ready for review.

**Comment by SubberTheFabulous** [comment]
Well, the amount of commits sure did get ugly, but should be fine, now.

**Comment by SubberTheFabulous** [comment]
!review

**Comment by skull132** [comment]
Commit count is largely irrelevant. We squash them all down into one tidy thing and it'll be fine.

**Pull request merged**